connections/walletconnect.js

```diff
@@ -0,0 +1,128 @@
+const EventEmitter = require("events");
+const WC = require("@walletconnect/browser");
+const WCQRCode = require("@walletconnect/qrcode-modal");
+const dev = process.env.NODE_ENV === "development";
+
+let WalletConnect;
+
+class WalletConnectConnection extends EventEmitter {
+  constructor(_WalletConnect, url, options) {
+    super();
+
+    this.qrcode =
+      typeof options.qrcode === "undefined" || options.qrcode !== false;
+
+    WalletConnect = _WalletConnect || WC;
+
+    setTimeout(() => this.create(url, options), 0);
```

Comment:
I feel a comment here would be justified. Perhaps something like

```
// make an async call to this.create. 
// As we don't care about waiting any amount of time but rather just want to perform this.create async,
// we use `0` as the second argument to setTimeout() 
```

This is maybe a bit overkill but I think could be useful.  Thoughts? ☺️👍